### ISSUE TITLE
 Lovász-Softmax loss

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#609]
# Reference
- [ ] [Project - LovaszSoftmax](http://bmax.im/LovaszSoftmax)
- [ ] [paper - The Lovász-Softmax loss: A tractable surrogate for the optimization of the intersection-over-union measure in neural networks](https://arxiv.org/pdf/1705.08790.pdf)
- [ ] [Github repo - LovaszSoftmax](https://github.com/bermanmaxim/LovaszSoftmax)
